scripts/variantstore/wdl/extract/build_docker.sh

```diff
@@ -1,13 +1,50 @@
-if [ $# -lt 1 ]; then
-    echo "USAGE: ./build_docker.sh [DOCKER_TAG_STRING] [OPTIONAL:LATEST]"
-    echo " e.g.: ./build_docker.sh \$(date -Idate)-alpine"
-    exit 1
+usage() {
+     echo "
+
+USAGE: ./build_docker.sh [--release|--branch]
+
+One of the options --release or --branch must be given, which determines the format of the Docker image tag.
+Release tags look like <ISO 8601 Date>-alpine, branch tags look like <ISO 8601 Date>-alpine-<short git hash>.
+
+e.g. 2023-06-06-alpine or 2023-06-06-alpine-ed338e48e
+"
+     exit 1
+}
+
```

Comment:
this indenting feels odd, but I guess as long as it works?!?!?

Reply:
ooh good catch